source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -89,7 +89,7 @@ namespace
 
     PvtOutput* findOutputOrThrow(const RtToken& name, PvtPrim* prim)
     {
-        PvtOutput* output = prim->getOutput(name);
+        PvtOutput* output = name.str().empty() ? prim->getOutput() : prim->getOutput(name);
```

Comment:
If no output is specified then use the appropriate getOutput() call. This was split into 2 and the second w/o an argument was never being called.